.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-10-05 19:12:07 UTC using RuboCop version 0.92.0.
```

Comment:
I re-generated this. The latest rubocop-shopify depends on ~> 0.89.0, hence the lower version.